ReactiveXComponent/Connection/IXCPublisher.cs

```diff
@@ -11,5 +12,7 @@ public interface IXCPublisher : IDisposable
         void SendEvent(StateMachineRefHeader stateMachineRefHeader, object message, Visibility visibility = Visibility.Public);
         List<MessageEventArgs> GetSnapshot(string stateMachine, int timeout = 10000);
         void GetSnapshotAsync(string stateMachine, Action<List<MessageEventArgs>> onSnapshotReceived);
+        List<string> GetXCApiNames(int timeout = 10000);
+        Stream GetXCApi(string apiFullName, int timeout = 10000);
```

Comment:
_GetXcApiList()_ and _GetXcApi()_ just like the commands provided by the WebSocket bridge.
And, as discussed, I think it's better to have these in **IXCSession** rather than in **IPublisher**.